### ISSUE TITLE
Improve login/sign up page

### ISSUE THREAD

**Issue by QAZIMAAZARSHAD** [QAZIMAAZARSHAD/Movie-Streaming-Website#13]
Improve design, add new features or totally redesign the login page.

**Comment by Godwindaniel10** [comment]
hi @QAZIMAAZARSHAD .
i will like to work on this issue

**Comment by QAZIMAAZARSHAD** [comment]
I will assign you this issue when the contribution starts then you can start working on this issue.

**Comment by Godwindaniel10** [comment]
ok on it..

**Comment by Dhanushreddy09** [comment]
Do you want to remove the background image under the form or you need it to be there?

**Comment by QAZIMAAZARSHAD** [comment]
Your wish, it's up to you. You can totally redesign the login page to make it better.

**Comment by Jagannath8** [comment]
@QAZIMAAZARSHAD  Can I work on this issue? I'm a participant of LGM Soc21.

**Comment by QAZIMAAZARSHAD** [comment]
I'll assign you this issue when the contribution period starts from 10th June, then you can start working on this issue.

Other participants are also willing to resolve this issue so If you want your pull request to be merged then your design should be better than the others.

**Comment by Dhanushreddy09** [comment]
are we supposed to display the result on the same page under form or we have to navigate to another page to display?

**Comment by QAZIMAAZARSHAD** [comment]
It's up to you if you want to display the result as a pop-up on the same page or navigate to another page, it's your wish.

**Comment by QAZIMAAZARSHAD** [comment]
Great work @Dhanushreddy09, I'll merge your pull request when the contribution period starts.

**Comment by Godwindaniel10** [comment]
@QAZIMAAZARSHAD Needless of me working on this issue right.. 🙂

**Comment by QAZIMAAZARSHAD** [comment]
Since many participants are willing to contribute or resolve a particular issue, I'll accept the best solution.
You can still work on the same issue, you can also create new issues if you find any mistake or chance of improvement in any solution.

I will start assigning issues and merging pull requests from 10th June.

**Comment by Godwindaniel10** [comment]
@QAZIMAAZARSHAD thats cool..

**Comment by Dhanushreddy09** [comment]
There is no way users can get back to the home page once they complete the registration, I am trying to relocate the URL to the home page but since javascript doesn't support relocation of URLs on submit type inputs, it's really difficult to cope up. Is there a suggestion I could rely on?

**Comment by QAZIMAAZARSHAD** [comment]
@Dhanushreddy09 this solution worked ->

<script>
      function DisplayResults() {
         alert("Registration successful");
         setTimeout(function(){document.location.href = "index.html"},100);
      };
    </script>

**Comment by Dhanushreddy09** [comment]
I came across this solution earlier but it isn't working for me. Anyways, I will commit it, however, I think we need to validate the user on the client-side. May I add javascript validation?

**Comment by QAZIMAAZARSHAD** [comment]
Yes, go-ahead

but don't make any pull requests before 10th June

**Comment by QAZIMAAZARSHAD** [comment]
> Hey mentors and project admin as the coding period for LGMSOC starts today could you assign the issue to me

@nisha331 this issue has already been resolved by @Dhanushreddy09 and will be closed soon.
Once that PR is merged you are welcome to open a new issue to improve the design or fix bugs if any.

**Comment by QAZIMAAZARSHAD** [comment]
@coding-geek21 this issue has already been resolved and will be closed soon.

You are always welcome to open a new issue to improve login page design again if you have some better idea.
Have patience till the other PR is merged.

**Comment by coding-geek21** [comment]
> @coding-geek21 this issue has already been resolved and will be closed soon.
> 
> You are always welcome to open a new issue to improve login page design again if you have some better idea.
> Have patience till the other PR is merged.

Sure thank you
